src/Main.elm

```diff
@@ -0,0 +1,25 @@
+module Main exposing (main)
+
+import Browser
+import Html exposing (button, div, text)
+import Html.Events exposing (onClick)
+
+main =
+  Browser.sandbox { init = 0, update = update, view = view }
```

Comment:
We’re grown ups, we can use `Browser.Application`

Reply:
Sure we can! But until we need all those things like URL change callbacks, a sandbox is enough, isn't it?

Reply:
Yup. But first thing we’d need from `Application` are flags for API keys.

Reply:
Alright. Let's merge this one and create another with displaying some list from API, in simple manner. The key will make sense from the start, then.